src/convexhull.jl

```diff
@@ -0,0 +1,30 @@
+"""
+Compute the upper part of the convex hull of a given set of 2D points 'pset' using the "Graham Scan" algorithm.
+
+*) Sort the points lexicographic
+
+*) Assess the last 3 points in 'upperHull'. If the the clockwise angle is <180° remove the second to last point.
+
+"""
+
```

Comment:
Check the other docstrings in the project. They follow a specific format with the name of the function followed by a description. Indentation matters.

Also there shouldn't be white space between the docstring and the function otherwise Julia won't be able to associate the docstring to the function in help mode.